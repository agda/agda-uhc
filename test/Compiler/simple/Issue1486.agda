module Issue1486 where

open import Common.Prelude

postulate
  QName : Set
{-# BUILTIN QNAME QName #-}

primitive
  primShowQName : QName -> String

main : IO Unit
main = putStrLn (primShowQName (quote main))
