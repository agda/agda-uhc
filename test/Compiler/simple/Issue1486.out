EXECUTED_PROGRAM

ret > ExitSuccess
out > Issue1486.main
out >
